angular_token_auth.js

```diff
@@ -297,13 +299,19 @@
             }
         };
 
-        //use cookies if available, otherwise try localstorage
-        if (storageMethods['cookie'].test() === true) {
-            return storageMethods['cookie'];
-        } else if (storageMethods['localStorage'].test()) {
-            return storageMethods['localStorage'];
+        if (MODULE_SETTINGS.STORAGE_METHOD && storageMethods[MODULE_SETTINGS.STORAGE_METHOD]) {
```

Comment:
No need for `MODULE_SETTINGS.STORAGE_METHOD &&` since it will return `undefined`, so `myObjectThatIsDefinitelyDefined[undefined]` will also return `undefined`. I think it's better to have a 1-to-1 relationship between the value of `MODULE_SETTINGS.STORAGE_METHOD` and finding a `storageMethods` key by that value, rather than also determine the truthy nature of the value.